src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -306,26 +290,61 @@ public HvdcShuntCompensatorsInfos getHvcLineShuntCompensators(UUID networkUuid,
 
     public List<String> getElementsIds(UUID networkUuid, String variantId, List<String> substationsIds, ElementType elementType) {
         switch (elementType) {
+            case SUBSTATION:
+                return getSubstationsIds(networkUuid, variantId);
```

Comment:
Filtering with substationsIds parameter if this parameter is not null ?

Reply:
It was already like that